docs/API.md

```diff
@@ -0,0 +1,26 @@
+# API
+
+## Socket events
+
+### Client
+
+Client-side API sends socket events to the server.
+
+sendToServer.js handles events dispatching.
+
+The serverObject provides methods to use :
+
+- `serverObject.sendNewUser()` returns a client ID (String)
+- `serverObject.sendDeleteUser()` returns a string telling that a client has disconnected.
+- `serverObject.sendMove(event)` returns movement events. Takes an event parameter corresponding to the executed movement.
+
+### Server
+
+Server-side API sends socket events to the client.
```

Comment:
From an API point of view, each object should document only the services it provides (methods with name, arguments + events with name, arguments) and maybe hints on how to use it, but not be directive of who sends what.